src/problems/RadMarshakDustPE/test_radiation_marshak_dust_and_PE.cpp

```diff
@@ -0,0 +1,311 @@
+//==============================================================================
+// TwoMomentRad - a radiation transport library for patch-based AMR codes
+// Copyright 2020 Benjamin Wibking.
+// Released under the MIT license. See LICENSE file included in the GitHub repo.
+//==============================================================================
+/// \file test_radiation_marshak_dust_and_PE.cpp
+/// \brief Defines a test Marshak wave problem with weak coupling between dust and gas.
+///
+
+#include "test_radiation_marshak_dust_and_PE.hpp"
+#include "AMReX.H"
+#include "QuokkaSimulation.hpp"
+#include "util/fextract.hpp"
+#include "util/valarray.hpp"
+
+struct StreamingProblem {
```

Comment:
Can we change the problem template name to something a bit less confusing, since this is not in fact a streaming problem?

Reply:
OK. Done.